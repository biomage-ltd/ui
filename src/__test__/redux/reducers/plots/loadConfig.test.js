import loadReducer from '../../../../redux/reducers/plots/loadConfig';

describe('loadConfig', () => {
  it('Config loaded', () => {
    const newState = loadReducer({}, {
      payload: {
        plotUuid: 'embeddingCategoricalMain',
        experimentId: '1234',
      },
    });
    expect(newState).toMatchSnapshot();
  });
});
