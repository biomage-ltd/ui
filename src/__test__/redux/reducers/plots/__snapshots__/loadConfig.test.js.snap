// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadConfig Config loaded 1`] = `
Object {
  "embeddingCategoricalMain": Object {
    "experimentId": "1234",
    "outstandingChanges": false,
  },
}
`;
